release_notes/42-release-notes.adoc

```diff
@@ -410,6 +410,17 @@ Red Hat has removed the *Use Container IAM role* option from {product-title-mana
 * In previous {product-title-short} versions, there was a mismatch in the number of CVEs reported by JSON output and table output. This issue is now fixed. (ROX-15277)
 * Previously, integrating {product-title-short} with Jira failed when you used an OAuth token or JWT while configuring the integration. This issue is fixed. (ROX-17992)
 
+[id="resolved-in-version-4201_{context}"]
+=== Resolved in version 4.2.1
+
+*Release date*: 2 October 2023
+
+* This release of {product-title-short} fixes the security vulnerability link:https://access.redhat.com/security/cve/CVE-2023-3899[CVE-2023-3899], a vulnerability in subscription-manager that allows local privilege escalation due to inadequate authorization.
+
+* This release also fixes the following issues:
+** A sensor panic can occur in version 4.2.0 if a cluster contains deployments with an invalid image reference, for example, `image: " "`, and delegated scanning is enabled for all registries.
+** A permission change in the {product-title-short} web portal with the 4.2.0 release was too broad and caused users with scoped roles to be unable to view certain portal pages.
```

Comment:
Here is a draft for your consideration.

* The minimum permissions in the {product-title-short} web portal to display 6 of the **sidebar links** was too strict in the 4.2.0 release and therefore has been reduced in the 4.2.1 release. The conditional display of buttons, links, and tabs **within** pages according to permission sets of user roles has been retained and will continue to increase in future releases. The clear visual cause-and-effect feedback helps platform administrators to decide which permissions resources to select for user roles according to their workflows in the web portal.

P.S. **scoped** roles could be read as relating to access scopes, yet the changes are for permission sets

Reply:
Less is more.